examples/Basics/user_callable.py

```diff
@@ -0,0 +1,176 @@
+
+
+### Other parameters
+
+# Two other boolean hyperparameters can be specified when creating a recommender object.
+# The first one allows the recommendation of points that were already recommended previously.
+# The second one allows the recommendation of points that have already been measured.
+# Per default, they are set to `True`.
+
+ALLOW_REPEATED_RECOMMENDATIONS = True
+ALLOW_RECOMMENDING_ALREADY_MEASURED = True
+
+### Creating the recommender object
+
+# To create the recommender object, each parameter described above can be specified as follows.
+# Note that they all have default values.
+# Therefore one does not need to specify all of them to create a recommender object.
+
+recommender = TwoPhaseMetaRecommender(
+    initial_recommender=INITIAL_RECOMMENDER,
+    recommender=SequentialGreedyRecommender(
+        surrogate_model=SURROGATE_MODEL,
+        acquisition_function=ACQ_FUNCTION,
+        allow_repeated_recommendations=ALLOW_REPEATED_RECOMMENDATIONS,
+        allow_recommending_already_measured=ALLOW_RECOMMENDING_ALREADY_MEASURED,
+        user_callable=callable_test,
+    ),
+)
+
+### Example Searchspace and objective parameters
+
+# We use the same data used in the [`campaign`](./campaign.md) example.
+
+dict_solvent = {
+    "DMAc": r"CC(N(C)C)=O",
+    "Butyornitrile": r"CCCC#N",
+    "Butyl Ester": r"CCCCOC(C)=O",
+    "p-Xylene": r"CC1=CC=C(C)C=C1",
+}
+dict_base = {
+    "Potassium acetate": r"O=C([O-])C.[K+]",
+    "Potassium pivalate": r"O=C([O-])C(C)(C)C.[K+]",
+    "Cesium acetate": r"O=C([O-])C.[Cs+]",
+    "Cesium pivalate": r"O=C([O-])C(C)(C)C.[Cs+]",
+}
+dict_ligand = {
+    "BrettPhos": r"CC(C)C1=CC(C(C)C)=C(C(C(C)C)=C1)C2=C(P(C3CCCCC3)C4CCCCC4)C(OC)="
+    "CC=C2OC",
+    "Di-tert-butylphenylphosphine": r"CC(C)(C)P(C1=CC=CC=C1)C(C)(C)C",
+    "(t-Bu)PhCPhos": r"CN(C)C1=CC=CC(N(C)C)=C1C2=CC=CC=C2P(C(C)(C)C)C3=CC=CC=C3",
+}
+
+solvent = SubstanceParameter("Solvent", data=dict_solvent, encoding="MORDRED")
+base = SubstanceParameter("Base", data=dict_base, encoding="MORDRED")
+ligand = SubstanceParameter("Ligand", data=dict_ligand, encoding="MORDRED")
+temperature = NumericalDiscreteParameter(
+    "Temperature", values=[90, 105, 120], tolerance=2
+)
+concentration = NumericalDiscreteParameter(
+    "Concentration", values=[0.057, 0.1, 0.153], tolerance=0.005
+)
+
+# We collect all parameters in a list.
+
+parameters = [solvent, base, ligand, temperature, concentration]
+
+# We create the searchspace and the objective.
+
+searchspace = SearchSpace.from_product(parameters=parameters)
+
+objective = SingleTargetObjective(target=NumericalTarget(name="yield", mode="MAX"))
+
+### Creating the campaign
+
+# The recommender object can now be used together with the searchspace and the objective as follows.
+
+campaign = Campaign(
+    searchspace=searchspace,
+    recommender=recommender,
+    objective=objective,
+)
+
+# This campaign can then be used to get recommendations and add measurements:
```

Comment:
It is not necessary to go into the details here. Just something like "We now run some loops of the recommendation process to demonstrate the use of the callable" should be sufficient.